addons/api/addon/generated/models/auth-method.js

```diff
@@ -221,6 +221,12 @@ export default class GeneratedAuthMethodModel extends BaseModel {
   })
   enable_groups;
 
+  @attr('boolean', {
+    for: 'ldap',
+    isNestedAttribute: true,
+  })
+  use_token_groups;
```

Comment:
good catch! 
Can we also add this field to the [mocks](https://github.com/hashicorp/boundary-ui/blob/b2b08929a3dbe9e4d9519a2844dce07c64c4a7d8/addons/api/mirage/generated/factories/auth-method.js#L48)?

Reply:
ooh yes! updated! thank you